### ISSUE TITLE
Enhancement - To evaluate predicates with this inspector.

### ISSUE THREAD

**Issue by FagunSafi** [mykola-mokhnach/Appium-iOS-Inspector#35]
We are enjoying this tool so much. However, now iOS Appium is giving us goosebumps with XPATH, we have to change our locator strategy to Predicates.

Any plan to enhance this tool to inspect iOS predicate strategy? If so, any ETA?

Appreciate your help.

**Comment by mykola-mokhnach** [comment]
I don't think it would be easy to implement such feature, but you are welcome to try.
